tests/__tests__/tsconfig-default.spec.ts

```diff
@@ -17,15 +17,6 @@ describe('get default ts config', () => {
     getTSConfig.cache.clear();
   });
 
-  it('should bail when no default tsconfig.json found', () => {
-    // there is no tsconfig file in that test module
-    ((path as any) as MockedPath).__setBaseDir('./tests/jestconfig-test');
-
-    expect(() => getTSConfig(mockJestConfig('jestconfig-test'))).toThrowError(
-      /unable to find ts configuration file/i,
-    );
-  });
-
```

Comment:
I think instead you should change the test here to ensure it loads defaults without bailing